doc/release-notes/4259-java-11.md

```diff
@@ -54,28 +61,43 @@ For installations that were previously running on Dataverse 4.x, the following J
 
 In domain.xml:
 
-Remove the following JVM options from the domain><configs><config name-"server-config"><java-config> element::
+Remove the following JVM options from the domain><configs><config name-"server-config"><java-config> element:
+
+`<jvm-options>-Djava.endorsed.dirs=/usr/local/payara5/glassfish/modules/endorsed:/usr/local/payara5/glassfish/lib/endorsed</jvm-options>`  
 
-  <jvm-options>-Djava.endorsed.dirs=/usr/local/payara5/glassfish/modules/endorsed:/usr/local/payara5/glassfish/lib/endorsed</jvm-options>
-  459a448
-  <jvm-options>-Djava.ext.dirs=${com.sun.aas.javaRoot}/lib/ext${path.separator}${com.sun.aas.javaRoot}/jre/lib/ext${path.separator}${com.sun.aas.instanceRoot}/lib/ext</jvm-options>
+`459a448`  
```

Comment:
Not sure what this is.

Reply:
I grabbed this from the original content in https://github.com/IQSS/dataverse/pull/7514#issue-557671297 

@qqmyers is this needed?

Reply:
Yeah, I'm fairly sure it may be a typo or maybe result from some merge conflict.

Reply:
removed in https://github.com/IQSS/dataverse/pull/7544/commits/cbda240f16e1520e3cfa6bd7ae8ab055523b540f

Reply:
...just part of a diff I think comparing line 459 in one file against 448 in the other. Shouldn't have been there.